Sources/Main/main.swift

```diff
@@ -1,29 +1,48 @@
 import Foundation
-import Routes
+import CobSpec
 import Server
-import Utility
-
-let utility = try CLIUtility()
-let port = utility.getPortNumber()
-let directory = utility.getDirectory()
-
-let files = ["data.txt", "requestLog.txt"]
-for file in files {
-    let filePath = NSURL.fileURL(withPathComponents: [file])
-    let path: String = filePath!.path
-    if (!FileManager.default.fileExists(atPath: path)) {
-        FileManager.default.createFile(atPath: path, contents: Data(), attributes: nil)
+
+public class Main {
+
+    public init(){}
+
+    public func prepareAppForServer() throws {
+        let utility = try CLIUtility()
+        let port = utility.getPortNumber()
+        let directory = utility.getDirectory()
+        createWritableFiles()
+
+        let routes = getRoutes()
+
+        if (directory == nil) {
+            print("Error: No Public Directory Found")
+        } else {
+            let server = Server(parser: Parser(directory: directory!), port: port, responder: Responder(), router: Router(routes: routes))
+            print("Your server is running on port: \(port) with the following directory: \(directory!)")
+            try server.startServer()
+        }
     }
-}
 
-let fourOhFour = FourOhFour()
-let router = Router(fourOhFour: fourOhFour)
+    private func createWritableFiles() {
+        let files = ["data.txt", "requestLog.txt"]
+        for file in files {
+            let filePath = NSURL.fileURL(withPathComponents: [file])
+            let path: String = filePath!.path
+            if (!FileManager.default.fileExists(atPath: path)) {
+                FileManager.default.createFile(atPath: path, contents: Data(), attributes: nil)
+            }
+        }
+    }
+    
+    private func getRoutes() -> [String: [String: Any]] {
+        let allRoutes = Routes.allValues
+        var routes: [String: [String: Any]] = ["":["":""]]
```

Comment:
Does this need to be `Any`? Why not find a way to say it's a `RouteHandler` or a `RouteAction`?

Reply:
I chose any given the structure of the data: 
`routes[route.rawValue] = ["routeHandler": route.routes, "allowedActions": route.routeActions]`
`routeHandler` is a Route Object
 `allowedActions` are Enums
Is there another way to use something other than `Any` in this instance? I couldn't think of anything so I differed to `Any`

Reply:
I think you're correct to use `Any`. I thought there was an easy way to declare a type alias as one or the other. ex: `typealias dictionaryValue = Route || AllowedAction`, but digging a little deeper it looks like I'm wrong.